README.md

```diff
@@ -49,6 +49,17 @@ curl https://raw.githubusercontent.com/Kha/elan/master/elan-init.sh -sSf | sh
 
 Alternatively, **any supported platform**: Grab the [latest release](https://github.com/Kha/elan/releases/latest) for your platform, unpack it, run it in a terminal, and follow the printed instructions.
 
+# Prerequisites
+
+On some systems `elan` will not work out of the box:
+
+* Windows 10
+   * You'll need a unix-like terminal.
+      * "git bash", available via [Git for Windows](https://gitforwindows.org/) works well, or
+      * the terminal from [MSYS2](https://www.msys2.org/), and then run `pacman -S unzip`
+   * [Visual C++ redistributable](https://www.microsoft.com/en-au/download/details.aspx?id=48145) (if you've already installed other software you likely have this).
+* macOS: You'll need to install `libgmp`. Install [Homebrew](https://brew.sh/), then run `brew install libgmp`.
```

Comment:
Perhaps add "to run Lean" after "libgmp" to clarify that this is not an issue with elan itself

Reply:
I've clarified this.